crate/server/src/core/operations/certify.rs

```diff
@@ -174,13 +175,15 @@ pub async fn certify(
                 AtomicOperation::Upsert((
                     issued_certificate_id,
                     issued_certificate,
+                    attributes.clone(),
                     Some(tags),
                     StateEnumeration::Active,
                 )),
                 // update the public key
                 AtomicOperation::UpdateObject((
                     public_key_owm.id.clone(),
-                    public_key_owm.object,
+                    public_key_owm.object.clone(),
+                    public_key_owm.object.attributes()?.clone(),
```

Comment:
avoid to clone the whole object by cloning the attributes first:
```rust
        let pk_own_obj_attributes = public_key_owm.object.attributes()?.clone();
        // return
        (
            issued_certificate_id.clone(),
            vec![
                // upsert the certificate
                AtomicOperation::Upsert((
                    issued_certificate_id,
                    issued_certificate,
                    attributes.clone(),
                    Some(tags),
                    StateEnumeration::Active,
                )),
                // update the public key
                AtomicOperation::UpdateObject((
                    public_key_owm.id.clone(),
                    public_key_owm.object,
                    pk_own_obj_attributes,
```

Reply:
thanks!